### PR TITLE
feat(api): setup routes

### LINKED ISSUE #18: Backend :: Implement routing
* Implement routing with dummy functions for starters

### PR DESCRIPTION
**Pull request by vanhanenjjv**

### CODE REVIEW

**Review by aarotoiv — APPROVED** [review]
Veri nais

File: api/src/routes/admin/quizzes/index.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+use worker::*;
+
+use crate::RequestData;
+
+pub async fn post(_req: Request, ctx: RouteContext<RequestData>) -> Result<Response> {
+    match ctx.data.admin_key {
+        Some(admin_key) => Response::ok(admin_key),
+        None => Response::error("Admin key is missing from the request", 400)
```
Comment (tominkki) [inline api/src/routes/admin/quizzes/index.rs]: Should we use rather 401 Unauthorized?

File: api/src/routes/admin/quizzes/submissions.rs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+use worker::*;
+
+use crate::RequestData;
+
+pub async fn get(_req: Request, ctx: RouteContext<RequestData>) -> Result<Response> {
+    match ctx.data.admin_key {
+        Some(admin_key) => Response::ok(admin_key),
+        None => Response::error("Admin key is missing from the request", 400)
```
Comment (tominkki) [inline api/src/routes/admin/quizzes/submissions.rs]: Same here

**Review by tominkki — CHANGES_REQUESTED** [review]

File: api/src/routes/admin/quizzes/index.rs (hunk 1 above)
Comment (vanhanenjjv) [reply api/src/routes/admin/quizzes/index.rs]: Yes. :)

File: api/src/routes/admin/quizzes/submissions.rs (hunk 2 above)
Comment (vanhanenjjv) [reply api/src/routes/admin/quizzes/submissions.rs]: Yes. :)

**Pull request merged**